src/test/java/com/google/cloud/genomics/dataflow/utils/PairGeneratorTest.java

```diff
@@ -0,0 +1,36 @@
+package com.google.cloud.genomics.dataflow.utils;
+
+import static org.junit.Assert.*;
```

Comment:
shift-control-O to "organize imports" and expand the * into the specific imports you need.